--- webrtc/src/api/audio_codecs/builtin_audio_decoder_factory.h
+++ webrtc/src/api/audio_codecs/builtin_audio_decoder_factory.h
@@ -21,7 +21,7 @@ namespace webrtc {
 // only need a subset of the codecs, consider using
 // CreateAudioDecoderFactory<...codecs listed here...>() or
 // CreateOpusAudioDecoderFactory() instead.
-rtc::scoped_refptr<AudioDecoderFactory> CreateBuiltinAudioDecoderFactory();
+RTC_EXPORT rtc::scoped_refptr<AudioDecoderFactory> CreateBuiltinAudioDecoderFactory();
 
 }  // namespace webrtc
 
