--- webrtc/src/sdk/BUILD.gn
+++ webrtc/src/sdk/BUILD.gn
@@ -1305,7 +1305,43 @@ if (is_ios || is_mac) {
           "@rpath/$output_name.framework/$output_name",
         ]
 
+        public_deps = [
+          "../third_party/abseil-cpp/absl/types:bad_optional_access",
+        ]
+
         deps = [
+          "../api:create_peerconnection_factory",
+          "../api:libjingle_peerconnection_api",
+          "../api:rtc_error",
+          "../api:transport_api",
+          "../api/crypto",
+          "../api/rtc_event_log:rtc_event_log_factory",
+          "../api/task_queue",
+          "../api/task_queue:default_task_queue_factory",
+          "../api/video_codecs:builtin_video_decoder_factory",
+          "../api/video_codecs:builtin_video_encoder_factory",
+          "../api/audio_codecs:builtin_audio_decoder_factory",
+          "../api/audio_codecs:builtin_audio_encoder_factory",
+          "../audio",
+          "../call",
+          "../common_audio",
+          "../common_video",
+          "../logging:rtc_event_log_api",
+          "../media",
+          "../modules",
+          "../modules/video_capture:video_capture_internal_impl",
+          "../p2p:rtc_p2p",
+          "../pc:libjingle_peerconnection",
+          "../pc:peerconnection",
+          "../pc:rtc_pc",
+          "../pc:rtc_pc_base",
+          "../rtc_base",
+          "../video",
+          "../api",
+          "../logging",
+          "../p2p",
+          "../pc",
+          "../stats",
           ":audio_objc",
           ":base_objc",
           ":default_codec_factory_objc",
